app/forms/idv/api_image_upload_form.rb

```diff
@@ -472,5 +472,9 @@ def store_failed_images(client_response, doc_pii_response)
     def image_resubmission_check?
       IdentityConfig.store.doc_auth_check_failed_image_resubmission_enabled
     end
+
+    def liveness_checking_required?
```

Comment:
Does this method add value?

Reply:
@soniaconnolly  - 👍🏿  ... it allows us to access liveness_checking_required for selfie validator

Reply:
i tried again to remove it and it worked 👍🏿